test/unit/selector.js

```diff
@@ -971,6 +971,39 @@ test("pseudo - form", function() {
 	extraTexts.remove();
 });
 
+test("pseudo - :(dis|en)abled, explicitly disabled", function() {
+	expect ( 2 );
+
+	t(
+		"Explicitly disabled elements",
+		"#enabled-fieldset :disabled",
+		[ "disabled-input", "disabled-textarea", "disabled-button",
+			"disabled-select", "disabled-optgroup", "disabled-option" ]
+	);
+
+	t(
+		"Enabled elements",
+		"#enabled-fieldset :enabled",
+		[ "enabled-input", "enabled-textarea", "enabled-button",
+			"enabled-select", "enabled-optgroup", "enabled-option" ]
+	);
```

Comment:
These needs to test `:disabled` & `:enabled` directly against fieldset elements as well: https://github.com/jquery/jquery/issues/2275#issuecomment-99955222